gems/smithy-client/lib/smithy-client/shapes/prelude_shapes.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Smithy
+  module Client
+    module Shapes
+      module PreludeShapes
```

Comment:
I think whether these are prelude shapes or not doesn't matter, they are shapes. I think we should include them where all the other shapes are defined rather than including this module always. Possibly these should be added to code generated output such that if we need defaults listed in prelude, they will change with service update and not runtime update. Can team discuss.

Reply:
see: https://github.com/smithy-lang/smithy-ruby/pull/221#discussion_r1896011649

for the team discussion result